src/Mvc/Mvc.Analyzers/src/Microsoft.AspNetCore.Mvc.Analyzers.csproj

```diff
@@ -15,7 +15,7 @@
     <Reference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" PrivateAssets="All" />
   </ItemGroup>
 
-  <Target Name="PopulateNuspec" BeforeTargets="GenerateNuspec" DependsOnTargets="Build">
```

Comment:
Why is this dependency removed?

Reply:
I just grabbed whatever was on @sebastienros PR to update the SDK. Honestly not sure.

Reply:
Filed https://github.com/aspnet/AspNetCore-Internal/issues/2116 to verify if the outcome's the same. Don't spend time investigating this if things build

Reply:
This was due to some build ordering issue with analyzers that @pakrym found in his initial investigation.